product/common-src/i18n/localeDataEn.ts

```diff
@@ -102,5 +102,12 @@ export const localeDataEn: ILocaleData = {
 
   COMMON_listingConnma: ', ',
 
-  defaultFileName: 'Untitled'
+  defaultFileName: 'Untitled',
+
+  CHECK_RULE: 'チェックルール',
```

Comment:
@ics-nishihara すみません、一度Approveしたのですが、こちらの内容を英語にしておきたいです。
また、変数名をほかの箇所と統一させて`大文字(カテゴリー)_小文字`としておきたいです。
```suggestion
  RULE_title: 'Check rule,
```

Reply:
@ics-nohara 
ありがとうございます。
変数名・英語表記を修正しました。